sopel/modules/wikipedia.py

```diff
@@ -223,7 +227,7 @@ def say_section(bot, trigger, server, query, section):
 
     snippet = mw_section(server, query, section)
     if not snippet:
-        bot.reply("Error fetching section \"{}\" for page \"{}\".".format(section, page_name))
+        bot.say('Error fetching section "{}" for page "{}".'.format(section, page_name))
```

Comment:
see https://github.com/sopel-irc/sopel/pull/2315/files#r920725745

```suggestion
        bot.say(f'Error fetching section "{section}" for page "{page_name}".')
```